build.gradle

```diff
@@ -224,6 +225,12 @@ project('spring-rabbit') {
 		}
 
 		compile ("ch.qos.logback:logback-classic:$logbackVersion", optional)
+
+		compile ("org.apache.logging.log4j:log4j-core:$log4j2Version", optional)
+		compile ("org.apache.logging.log4j:log4j-jcl:$log4j2Version", optional)
```

Comment:
Yeah. Agree. But I thought that my next comment in the email stream should explain the reason.
Sorry for confuse.